files/default/sql/ddl/updates/undo/2.1.0__undo.sql

```diff
@@ -105,4 +105,6 @@ SET SQL_SAFE_UPDATES = 1;
 ALTER TABLE `hopsfs_training_dataset` DROP FOREIGN KEY `hopsfs_td_conn_fk`,
     DROP COLUMN `connector_id`;
 
-DROP TABLE `feature_store_connector`;
\ No newline at end of file
+ALTER TABLE `on_demand_feature_group` DROP COLUMN `data_format`, DROP COLUMN `path`;
```

Comment:
What about the ```MODIFY `query` VARCHAR(11000), ``` shouldn't that one also be undone? e.g. set the NOT NULL constraint again.

Reply:
done